README.md

```diff
@@ -189,6 +187,8 @@ whom they elect at their own discretion.
 
 ### TSC Emeriti
 
+* [joshgav](https://github.com/joshgav) -
```

Comment:
This list is sorted by GitHub username. Can you move it to the correct place?

Reply:
fixed!